### ISSUE TITLE
Merge cells & Set autofilter

### ISSUE THREAD

**Issue by AndreeaMb** [aVadim483/fast-excel-writer#69]
When I try to generate an excel file, and I use  **mergeCells()** **and setAutofilter()** togheter, there is a conflict in the file.

This is my code(I try to merge some cells in the first row, and to add the autofilters for the second row):

```
....
$sheet->setAutofilter(2);

if($this->current_tab == 0) {
    $sheet->mergeCells("A1:E1");
}
....
```

The excel file is generated without errors, but when I try to open the file, I can not and I receive this alert:

_Alert
We found a problem with some content in ’xxxxx.xlsx’. Do you want us to try to recover as much as we can? If you trust the source of this workbook, click Yes._

Is there a way to use these methods togheter?

**Comment by aVadim483** [comment]
Well, I will look into this problem

**Comment by aVadim483** [comment]
fixed in version 5.6.1

**Comment by AndreeaMb** [comment]
Thank you!
